audio_windows.go

```diff
@@ -0,0 +1,41 @@
+package engi
+
+import (
+	"log"
+
+	"github.com/paked/engi/ecs"
+)
+
+const (
+	defaultHeightModifier float32 = 1
+)
+
+// AudioComponent is a Component which is used by the AudioSystem
```

Comment:
This should be defined in a separate `audio.go`, also... Probably should make this file be ommitted from a non-windows build.